<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="丹哥" ignored="0" name="AutomationFrameworks" tests="1" failures="1" timestamp="04 七月 2018 04:38:14 GMT" time="5.096" errors="0">
  <testcase name="aaa" time="5.052" classname="testPractice.Test000056">
    <failure type="java.lang.AssertionError" message="expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [2]at com.test.utill.AssertFunction.assertEquals(AssertFunction.java:14)at testPractice.Test000056.aaa(Test000056.java:16)... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- aaa -->
</testsuite> <!-- AutomationFrameworks -->
